crates/rpc-types-anvil/src/lib.rs

```diff
@@ -0,0 +1,164 @@
+#![doc = include_str!("../README.md")]
+#![doc(
+    html_logo_url = "https://raw.githubusercontent.com/alloy-rs/core/main/assets/alloy.jpg",
+    html_favicon_url = "https://raw.githubusercontent.com/alloy-rs/core/main/assets/favicon.ico"
+)]
+#![warn(
+    missing_copy_implementations,
+    missing_debug_implementations,
+    unreachable_pub,
+    clippy::missing_const_for_fn,
+    rustdoc::all
+)]
+#![allow(missing_docs)]
+#![cfg_attr(not(test), warn(unused_crate_dependencies))]
+#![deny(unused_must_use, rust_2018_idioms)]
+#![cfg_attr(docsrs, feature(doc_cfg, doc_auto_cfg))]
+
+use alloy_primitives::{TxHash, B256, U256};
+use serde::{Deserialize, Deserializer, Serialize};
+use std::collections::BTreeMap;
+
+/// Represents the params to set forking which can take various forms
+///  - untagged
+///  - tagged `forking`
```

Comment:
```suggestion
///  - untagged
///  - tagged forking
```

maybe we should clarify what tagged vs untagged means? i assume its just one is w block number the other without?

Reply:
yeah, could be nested object or not